### ISSUE TITLE
Update file tree when external content is added

### ISSUE THREAD

**Issue by aranega** [silverwind/droppy#252]
Hi again,

Everything is running fine with droppy on my raspberry, I really love it! 

However, there is one thing that seems a little bit strange: I have a dedicated file on my droppy repository where I put youtube videos. This file is read once an hour a cron job is launched and all the audio from the link in the file are fetched to the repository. Currently, once the audio files are fetched, droppy does not seems to see that files have been added, the only solution I found is to "force" it to refresh the repository content by "restarting" the "droppy.service". 

Is there another solution ?  Perhaps it's an issue with my config.

**Comment by silverwind** [comment]
Try setting a `pollingInterval` in the config. If the files are on an external drive, the change detections don't work reliably. Something like 3000 might be a good start, files should show up with a few seconds delay.

**Comment by aranega** [comment]
Indeed, my files are on an external drive (an old one). I will try with the `pollingInterval`, I hope this will fix the issue :) 

I close this ticket as the issue is definitely related to my config, I have to find the good parameters so everything work fine. 

Thank you!

**Comment by aranega** [comment]
Just for the record, I finally found the configuration that works for me:

1. I set the `pollingInterval` to 30000 so the CPU is not too busy all the time.
1. Instead of directly fetching files into the exernal drive folder:
 * I copy them in `/tmp` first (as intermediate folder) --> the work is performed fast enough
 * I move the file from `/tmp` to my external drive
 * I perform a little `sync` just to be sure ;)

It works like a charm now. Thank you again for the tips.
